envoy/api/v2/filter/network/tcp_proxy.proto

```diff
@@ -109,6 +109,15 @@ message TcpProxy {
 
   // TCP Proxy filter configuration using deprecated V1 format. This is required for complex
   // routing until filter chain matching in the listener is implemented.
+  //
+  // .. attention::
+  //
+  //   Using this field will lead to `problems loading the configuration
+  //   <https://github.com/envoyproxy/envoy/issues/2441>`_. If you
+  //   want to configure the filter using v1 config structure, please make this
+  //   field a boolean with value ``true`` and configure via the opaque ``value`` field
+  //   like is suggested in the filter `README
+  //   <https://github.com/envoyproxy/data-plane-api/blob/master/envoy/api/v2/filter/README.md>`_.
   DeprecatedV1 deprecated_v1 = 6;
```

Comment:
Do you want to tag this explicitly as deprecated, i.e. as with https://github.com/envoyproxy/data-plane-api/blob/a66448b203e3a4f8ebbea0a5b7beede5c25640f6/envoy/config/ratelimit/v2/rls.proto#L23, or is this not formally deprecated yet?

Reply:
I guess at this point we should deprecated it, as it can't actually be used.

Reply:
I think it is fair to deprecate, given it is doa anyway.